dafter/cli.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
 
         if not (os.path.exists(DATASETS_FOLDER)):
             os.makedirs(DATASETS_FOLDER)
-            print("Folder for datasets not present, creating it at '%s'" % DATASETS_FOLDER)
+            print("Folder for datasets not present, creating it at {}".format(DATASETS_FOLDER))
```

Comment:
I approve the change. However, this won't be supported by python 2, if I not wrong.

Reply:
I just tried it and it works (`python2.7`). It seems it does work since `python2.6`: https://docs.python.org/fr/2.7/library/stdtypes.html#str.format

Reply:
Oh, yes I confound it with [Literal String Interpolation](https://www.python.org/dev/peps/pep-0498/).